Examples/UIExplorer/js/NativeAnimationsExample.js

```diff
@@ -49,7 +49,7 @@ class Tester extends React.Component {
       this.current && this.props.reverseConfig ? this.props.reverseConfig : this.props.config
     );
     this.current = this.current ? 0 : 1;
-    const config = {
+    const config: Object = {
```

Comment:
This does a bunch of metaprogramming and would be a pain to properly typecheck so just cast to Object.